org-formation/600-access/_tasks.yaml

```diff
@@ -214,6 +214,21 @@ StackArmorReadOnlyAccess:
       - arn:aws:iam::aws:policy/AmazonEC2ReadOnlyAccess
       - arn:aws:iam::aws:policy/AmazonVPCReadOnlyAccess
 
+# Setup SynapseProd cross-account access the BedRock in SynapseLlmProd
+# https://repost.aws/knowledge-center/bedrock-invoke-with-cross-account
+SynapseProdBedrockFullAccess:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.2.10/templates/IAM/cross-account-access.yaml
+  StackName: synapseprod-bedrock-full-access
+  DefaultOrganizationBinding:
+    Account: !Ref SynapseLlmProdAccount
+    Region: us-east-1
+  Parameters:
+    PrincipalArns:
+      - arn:aws:iam::325565585839:root # SynapseProd AWS account ID
```

Comment:
do you think you'll ever allow other accounts access to Synapse LLM?

Reply:
The idea now is to have separate BedRock accounts.